### ISSUE TITLE
Remove dependence on VA's starter plugins for Gradle

### ISSUE THREAD

**Issue by yoomlam** [department-of-veterans-affairs/abd-vro#1514]
Our Gradle build configuration requires VA's starter plugins (e.g., `gov.va.starter:plugins` and `gov.va.starter:cookiecutter`). Accessing the plugins require a GitHub PAT and the GitHub user to be part of the VA org.
As a result, this prevents the VRO open source repo from being built unless the user is part of the VA org. This also causing (intermittent?) [SecRel errors](https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/4821046623/jobs/8586969557#step:15:477) -- [notified SecRel team](https://dsva.slack.com/archives/C03UA9MV1EH/p1682610218938419?thread_ts=1681408060.901649&cid=C03UA9MV1EH).

One option is to copy the plugins into our repo.
```
> ls ~/.gradle/caches/modules-2/files-2.1/gov.va.starter
cookiecutter/
error-handling/
exceptions/
health/
open-api/
plugins/
starter-bom/
tracing/
```

**Comment by msnwatson** [comment]
Also, a note that some of these plugins may need to be re-evaluated entirely such as https://github.com/palantir/gradle-docker as the repo makes a disclaimer that the plugin is now defunct.

**Comment by yoomlam** [comment]
PRs merged: #1549, #1557
